class PyfortuneException(Exception):
    pass

class LoginFailureException(PyfortuneException):
    pass
